src/viam/app/data_client.py

```diff
@@ -806,6 +926,7 @@ def create_filter(
         end_time: Optional[datetime] = None,
         tags: Optional[List[str]] = None,
         bbox_labels: Optional[List[str]] = None,
+        dataset_id: Optional[str] = None,
     ) -> Filter:
         warnings.warn("DataClient.create_filter is deprecated. Use AppClient.create_filter instead.", DeprecationWarning, stacklevel=2)
```

Comment:
(opt) A bit of a flyby, but can we correct this message? The proper create_filter exists in `utils`, not `AppClient`.